title1 = ("Distinct Roles for FOXP3+ and FOXP3− CD4+ T Cells in Regulating Cellular Immunity to Uncomplicated and Severe Plasmodium falciparum Malaria (2009)")
example1 = '''Abstract

Failure to establish an appropriate balance between pro- and anti-inflammatory immune responses is believed to contribute to pathogenesis of severe malaria. To determine whether this balance is maintained by classical regulatory T cells (CD4+ FOXP3+ CD127−/low; Tregs) we compared cellular responses between Gambian children (n = 124) with severe Plasmodium falciparum malaria or uncomplicated malaria infections. Although no significant differences in Treg numbers or function were observed between the groups, Treg activity during acute disease was inversely correlated with malaria-specific memory responses detectable 28 days later. Thus, while Tregs may not regulate acute malarial inflammation, they may limit memory responses to levels that subsequently facilitate parasite clearance without causing immunopathology. Importantly, we identified a population of FOXP3−, CD45RO+ CD4+ T cells which coproduce IL-10 and IFN-γ. These cells are more prevalent in children with uncomplicated malaria than in those with severe disease, suggesting that they may be the regulators of acute malarial inflammation.
Go to:
Author Summary

While Tregs have been implicated in regulation of the immune response to chronic infections, their potential in determining disease outcome in acute infections is unclear. In this study we have found that Tregs are unable to control the florid inflammation during acute, severe P. falciparum malaria infections, suggesting that this component of the immunoregulatory arsenal may be rapidly overwhelmed by virulent infections. Further, we identified, for the first time in an acute human infection, a population of IL-10-producing Th-1 effector cells and found that IL-10-producing Th-1 cells were associated with development of uncomplicated as opposed to severe malaria, leading us to suggest that such “self-regulating” Th-1 cells may contribute to clearing malaria infections without inducing immune-mediated pathology. In addition, we found evidence that malaria-induced Tregs may limit the magnitude of malaria-specific memory responses detectable 28 days later, which may reduce the risk of immune-mediated pathology upon reinfection and may explain how immunity to severe disease can be gained after as little as one or two infections. We conclude that vaccines designed to induce cell-mediated responses should be assessed for their ability to induce IL-10 producing Th-1 cells and Tregs.
Go to:
Introduction

The clinical spectrum of P. falciparum infection ranges from asymptomatic parasite carriage to a febrile disease that may develop into a severe, life-threatening illness. The factors that determine disease severity are not completely understood but are likely to include both parasite and host components [1]–[3]. Ultimately, the interplay between the parasite and the immune response likely determines the outcome of the infection [4]. Although sterile immunity - completely preventing re-infection - is hardly ever seen and protection against clinical symptoms of uncomplicated disease is only acquired after repeated infections [5], immunity to severe disease and death may be acquired after as few as one or two infections [6] suggesting that different immune mechanisms underlie these different levels of immunity.

While there is a growing consensus that killing of malaria parasites or malaria-infected red blood cells requires the synergistic action of antibodies and cell-mediated immune responses [5],[7], the mechanisms conferring protection against severe disease are less clear. Given that pathology of severe disease has repeatedly been linked to sustained and/or excessive inflammatory responses [4], acquiring the ability to regulate these responses adequately may be a key determinant of immunity that protects against severe disease [8]. Thus, while an early inflammatory response is needed to control parasite replication in human P. falciparum malaria [9]–[11], excessive levels of pro-inflammatory cytokines such as TNF-α [12]–[15], IFN-γ, [16],[17], IL-1β and IL-6 [14],[18],[19] are associated with severe pathology. Conversely, low levels of regulatory cytokines such as TGF-ß have been associated with acute [20] and severe malaria [21],[22], a relative deficiency in IL-10 was seen in those who succumbed to severe malaria [23], significantly lower ratios of IL-10 to TNF-α were found in patients with severe malarial anaemia [24],[25], and high ratios of IFN-γ, TNF-α and IL-12 to TGF-β or IL-10 were associated with decreased risk of malaria but increased risk of clinical disease in those who became infected [26]. In summary, therefore, immunity against severe malaria may depend upon the host's ability to regulate the magnitude and timing of the cellular immune response, allowing the sequential induction of appropriate levels of inflammatory- and anti-inflammatory cytokines at key stages of the infection.

Given these associations between severe disease and exacerbated immune pathology, a number of studies have explored the role of CD4+CD25hiFOXP3+CD127−/lo regulatory T cells (Tregs) in determining the outcome of malaria infection. Induced and/or activated in response to malaria infection [27], Tregs may be beneficial to the host in the later part of the infection - when parasitaemia is being cleared - by down-regulating the inflammatory response and thereby preventing immune-mediated pathology. On the other hand, if Tregs mediate their suppressive effects too early, this could hamper the responses required for initial control of parasitaemia, permitting unbridled parasite growth which may also lead to severe disease. Malaria-specific induction of Tregs has been observed in a variety of experimental malaria infections in mice [28]–[31], but their role in preventing severe malarial pathology is unclear. Thus, in BALB/c mice infected with a lethal strain of P. yoelii, ablation of Treg activity by depletion of CD25+ cells either allowed mice to control parasitaemia and survive [32] or had no impact on the course of disease [33]. Depleting CD25+ cells of BALB/c mice infected with either P. berghei NK65 [34] or P berghei ANKA [35] reduced neither parasitaemia nor mortality, but increased the severity of symptoms in the diseased mice, suggesting at least some benefit from Tregs in this model. Rather oddly, infection of CD25+ T cell-depleted BALB/c mice with P. chabaudi adami DS led to increased parasitaemia and more severe anaemia [30]. Finally, CD25+ T cell depletion around the time of parasite inoculation reduced the incidence of experimental cerebral malaria in C57BL/6 mice infected with P. berghei ANKA in two independent studies [29],[31], but not when CD25+ T cells were depleted 30 days prior to infection [31]. Whilst various explanations have been offered for these discrepant results, including differences in the various strains of mice and parasites employed, the microbial microenvironment in which the mice are kept, and the precise CD25 depletion protocols employed, these studies are currently not very helpful when trying to understand the role of Tregs in human malaria infections.

Malaria naïve individuals undergoing experimental P. falciparum sporozoite infection showed an increase in FOXP3 mRNA expression and expansion of Tregs 10 days after infection; Treg induction correlated with high circulating levels of TGF-β, low levels of pro-inflammatory cytokines and rapid parasite growth [27] suggesting - but not proving - that Treg activation early in infection may inhibit the development of effective cellular immunity. More recently, we have observed that Treg populations appear to be transiently expanded and activated during the malaria transmission season in individuals from a malaria endemic community [36], again suggesting that naturally acquired malaria infection can drive the expansion and activation of Tregs. However, although Tregs have been implicated in IL-10-mediated down-regulation of Th1-like responses in the placenta of malaria-infected women [37] and reduced Treg frequencies and function have been linked to enhanced anti-parasite immunity in certain ethnic groups in West Africa [38] the potential for Tregs to influence the clinical outcome of malaria infections is still unclear.

To investigate the role of Tregs during clinical malaria infection, we have compared cellular immune responses of children with either severe or uncomplicated malaria. Interestingly, although we did not observe any significant differences in Treg numbers or function between severe and uncomplicated malaria cases, our data do indicate that malaria-induced Tregs may limit the magnitude of malaria-specific Th1 memory responses and thus moderate pro-inflammatory responses to subsequent infections, providing a possible explanation for the very rapid acquisition of immunity to severe malaria. Moreover, we have identified a population of FOXP3−, CD45RO+, CD4+ T cells which co-produce IL-10 and IFN-γ and which are more prevalent in children with uncomplicated malaria than in those with severe disease. We suggest that these IL-10 producing effector T cells may contribute to clearing malaria infection without-inducing immune-mediated pathology.
Go to:
Results

Immune responses of 59 Gambian children with severe P. falciparum malaria were compared with those of 65 children with uncomplicated clinical malaria and with 20 healthy (control) children of similar age and recruited from the same study area at the same time (Table 1). On admission, only 12 (9.4%) patients had a white blood cell count (WBC) above the age-specific norm and there was no significant difference in median WBC count between uncomplicated and severe cases, suggesting that few if any children had a concomitant systemic bacterial infection. No difference was observed in the differential WBC between the two groups. As expected, [39]–[41] numbers of lymphocytes, CD3+ and CD4+ T cells were significantly lower during the acute disease than during convalescence in both the severe and the uncomplicated groups (Table 1A). Parasite density on admission was two-fold higher in patients with severe malaria than in those with uncomplicated malaria; severely ill children also had significantly lower hemoglobin levels and were on average 2.4 years younger than children with uncomplicated disease. The number of P. falciparum clones per clinical isolate ranged from 1 to 4 (with an overall mean of 2 (CI95%: 1.8–2.1)), and – as has been observed previously [42] - did not differ significantly between the three groups (p = 0.3, Table 1B). Other factors potentially confounding immune responses, such as the degree of malnutrition or intestinal helminth infections were of similarly low prevalence in both severe and uncomplicated malaria cases and were not associated with severity of disease (Table 1B).
'''

title2 = "Are pangolins the intermediate host of the 2019 novel coronavirus (SARS-CoV-2)? (2020)"
example2 = '''
Abstract

The outbreak of a novel corona Virus Disease 2019 (COVID-19) in the city of Wuhan, China has resulted in more than 1.7 million laboratory confirmed cases all over the world. Recent studies showed that SARS-CoV-2 was likely originated from bats, but its intermediate hosts are still largely unknown. In this study, we assembled the complete genome of a coronavirus identified in 3 sick Malayan pangolins. The molecular and phylogenetic analyses showed that this pangolin coronavirus (pangolin-CoV-2020) is genetically related to the SARS-CoV-2 as well as a group of bat coronaviruses but do not support the SARS-CoV-2 emerged directly from the pangolin-CoV-2020. Our study suggests that pangolins are natural hosts of Betacoronaviruses. Large surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. In addition to conservation of wildlife, minimizing the exposures of humans to wildlife will be important to reduce the spillover risks of coronaviruses from wild animals to humans.
Author summary

Recently, a novel coronavirus, SARS-CoV-2, caused a still ongoing pandemic. Epidemiological study suggested this virus was associated with a wet market in Wuhan, China. However, the exact source of this virus is still unknown. In this study, we attempted to assemble the complete genome of a coronavirus identified from two groups of sick Malayan pangolins, which were likely to be smuggled for black market trade. The molecular and evolutionary analyses showed that this pangolin coronavirus we assembled was genetically associated with the SARS-CoV-2 but was not likely its precursor. This study suggested that pangolins are natural hosts of coronaviruses. Determining the spectrum of coronaviruses in pangolins can help understand the natural history of coronaviruses in wildlife and at the animal-human interface, and facilitate the prevention and control of coronavirus-associated emerging diseases.


Introduction

In December 2019, there was an outbreak of pneumonia with an unknown cause in Wuhan, Hubei province, China, with an epidemiological link to the Huanan Seafood Wholesale Market, a local live animal and seafood market. Clinical presentations of this disease greatly resembled viral pneumonia. Through deep sequencing on the lower respiratory tract samples of patients, a novel coronavirus named the 2019 novel coronavirus was identified [1], the name of which was then determined as SARS-CoV-2. This virus has spread to all provinces across China and more than 200 additional countries. As of April 11, 2020, the epidemic has resulted in 83,400 laboratory confirmed cases, 3,349 of which were fatal in China, while there were 1,643,047 laboratory confirmed cases and 101,507 deaths in other countries. The global toll of new cases and deaths is still increasing sharply.

To effectively control the disease and prevent new spillovers, it is critical to identify the animal origin of this newly emerging coronavirus. In the Wuhan wet market, high viral loads were reported in environmental samples. However, a variety of animals, including wildlife, were sold in this market, and the daily number and species of animals were very dynamic. Therefore, it remains unclear which animals initiated the first infections.

Coronaviruses usually cause respiratory and gastrointestinal tract infections and are genetically classified into four major genera: Alphacoronavirus, Betacoronavirus, Gammacoronavirus, and Deltacoronavirus. The former two genera primarily infect mammals, whereas the latter two predominantly infect birds [2]. In addition to SARS-CoV-2, other members of the Betacoronavirus genus caused the 2003 SARS (severe acute respiratory syndrome) outbreaks and the 2012 MERS (Middle East respiratory syndrome) outbreaks in humans [3, 4]. SARS-CoV and MERS-CoV are of bat origin, but both coronaviruses had an intermediate host: palm civets for SARS-CoV [5] and dromedary camels for MERS-CoV [6].

Approximately 30,000 base pairs in the coronavirus genome code for up to 11 proteins, including the surface glycoprotein Spike (S) protein binds to receptors on the host cell, which initiates virus infection. Different coronaviruses can use distinct host receptors due to structural variations in the receptor binding domains of their virus S protein. SARS-CoV uses angiotensin-converting enzyme 2 (ACE2) as one of the main receptors [7] with CD209L as an alternative receptor [8], whereas MERS-CoV uses dipeptidyl peptidase 4 (DPP4, also known as CD26) as the primary receptor. A recent study demonstrated that SARS-CoV-2 uses the SARS-CoV receptor ACE2 for entry and the serine protease TMPRSS2 for S protein priming [9].

Soon after the release of the SARS-CoV-2 genome, a scientist released a full genome of a coronavirus, Bat-CoV-RaTG13, from the bat species Rhinolophus affinis, which was colonized in Yunan province, nearly 2,000 km away from Wuhan. Bat-CoV-RaTG13 was 96% identical at the whole genome level to the SARS-CoV-2, suggesting the SARS-CoV-2 could be of bat origin [1]. However, because direct human-bat contact is rare, it seems to be more likely that the spillover of SARS-CoV-2 to humans from an intermediate host rather than directly from bats, as was the cases with both SARS-CoV and MERS-CoV.

The goal of this study was to determine the genetic relationship between a coronavirus from two groups of sick pangolins and SARS-CoV-2, and to assess whether pangolins could be potential intermediate hosts of SARS-CoV-2.
Results

In March and July of 2019, we detected Betacoronaviruses in three individuals from two sets of smuggled Malayan pangolins (Manis javanica) (n = 27) that were intercepted by Guangdong customs [10]. All three animals suffered from serious respiratory disease and failed to be rescued by the Guangdong Wildlife Rescue Center [10] (S1 Table). Through metagenomic sequencing and de novo assembling, we recovered 38 contigs ranging from 380 to 3,377 nucleotides, and the nucleotide sequence identity among the contigs from these three samples were 99.54%. Thus, we pooled sequences from three samples and assembled the draft genome of this pangolin origin coronavirus. After that, gap filling with amplicon sequencing was conducted to obtain a nearly full genome sequence. This pangolin-CoV-2020 genome (Genbank No.: MT121216) was found to be comprised of 29,521 nucleotides.

Strikingly, genomic analyses suggested the pangolin-CoV-2020 has a high identity with both SARS-CoV-2 and Bat-CoV-RaTG13, the proposed origin of SARS-CoV-2 (Fig 1A, S2 Table). The nucleotide sequence identity between pangolin-CoV-2020 and SARS-CoV-2 was 90.32%, whereas the protein sequence identity for individual proteins can be up to 100% (Table 1; Table 2). The nucleotide sequence identity between pangolin-CoV-2020 and Bat-CoV-RaTG13 was 90.24%, while that for the corresponding regions between SARS-CoV-2 and Bat-CoV-RaTG13 was 96.18% (Table 1, S1 Table).

On the other hand, unlike RBD, the nucleotide and amino acid sequence identity of NTD (N-terminal domain) were only 66.2% and 63.1% identical between pangolin-CoV-2020 and SARS-CoV-2. However, a loci Arg408 from the RBD core of SARS-CoV-2 could form a hydrogen bond with human ACE2 was conserved in pangolin-CoV-2020 (Fig 3). Both pangolin-CoV-2020 and Bat-CoV-RaTG13 lack an S1/S2 cleavage site (~680–690 aa) whereas SARS-CoV-2 possesses (Fig 3).

Genomic analyses suggested sequence similarities were not homogeneous across the S genes of pangolin-CoV-2020, SARS-CoV-2, Bat-CoV-ZXC21 and Bat-CoV-ZC45. For example, the first S region (i.e., nucleotides 1–1200) of pangolin-CoV-2020 has a higher nucleotide identity to two bat viruses (Bat-CoV-ZXC21 and Bat-CoV-ZC45) than SARS-CoV-2 and Bat-CoV-RaTG13, whereas the remaining S gene of pangolin-CoV-2020 is opposite (Fig 2A). These results suggest that a recombination event could have occurred during the evolution of these coronaviruses.

Phylogenetic analyses suggested that the S genes of pangolin-CoV-2020, SARS-CoV-2 and three bat origin coronaviruses (Bat-CoV-RaTG13, Bat-CoV-ZXC21, and Bat-CoV-ZC45) were genetically more similar to each other than other viruses in the same family (Fig 2B). The S gene of Bat-CoV-RaTG13 was genetically closer to pangolin-CoV-2020 than Bat-CoV-ZXC21 and Bat-CoV-ZC45. Similar tree topologies were observed for the encoding ORFs of RNA-dependent RNA polymerase (RdRp gene) and other genes (S1–S3 Figs). At the genomic level, SARS-CoV-2 was also genetically closer to Bat-CoV-RaTG13 than pangolin-CoV-2020 (Fig 1B).
Discussion

In this study, we assembled the genomes of coronaviruses identified in sick pangolins and our results showed that pangolin-CoV-2020 is genetically associated with both SARS-CoV-2 and a group of bat coronaviruses. There is a high sequence identity between pangolin-CoV-2020 and SARS-CoV-2. However, phylogenetic analyses and a special amino acid sequence in the S gene of SARS-CoV-2 did not support the hypothesis of SARS-CoV-2 arising directly from the pangolin-CoV-2020.

It is of interest that the genomic sequences of coronaviruses detected from two batches of smuggled pangolins intercepted by different customs at different dates were all be associated with bat coronaviruses. In addition, the genetic identity of coronavirus contigs assembled in each animal was extremely high (99.54%). The reads from the third pangolin acquired in July 2019 were relatively less abundant than those from the two pangolin samples acquired in March 2019. Although it is unclear whether coronaviruses in these two batches of smuggled pangolins had the same origin, our results indicated that the pangolins can be a natural host for Betacoronaviruses, which could be enzootic in pangolins.

All three exotic pangolins detected with Betacoronaviruses were sick with serious respiratory diseases and failed to be rescued. However, these pangolins were very stressful in the transportation freight when being intercepted by the customs. It is unclear whether this coronavirus is a common virus flora in the respiratory tract of pangolins. Nevertheless, the pathogenesis of this coronavirus in pangolins remains to be elucidated.

Phylogenetic trees suggested that Bat-CoV-RaTG13 was more genetically close to SARS-CoV-2 at both individual gene and genomic sequence level compared with the genomic sequence of pangolin-CoV-2020 assembled in this study. Recombination analysis showed that S gene of pangolin-CoV-2020 might be constructed by fragment from Bat-CoV-ZC45 or Bat-CoV-ZXC21 and fragment from Bat-CoV-RaTG13. Interestingly, the cleavage site between S1 and S2 in SARS-CoV-2 had multiple insertions (i.e. PRRA), compared with those of Bat-CoV-RaTG13 and pangolin-CoV-2020, which may result from an additional recombination event. A new study reported a novel bat-derived coronavirus (RmYN02) identified from a metagenomics analysis of samples from 227 bats collected from the Yunnan province in China between May and October of 2019. Although RmYN02 showed a relatively low nucleotide sequence identity (93.3%) to SARS-CoV-2, it had a similar manner of the insertion of multiple amino acids at the junction site of the S1 and S2 subunits of the S protein as SARS-CoV-2, providing strong evidence that such insertion events can occur in nature [11]. Thus, these data suggest that SARS-CoV-2 originated from multiple naturally occurring recombination events among viruses present in bats and other wildlife species.

The S protein of coronaviruses binds to host receptors via RBDs and plays an essential role in initiating viral infection and determining host tropism [2]. A prior study suggested that SARS-CoV-2 and SARS-CoV bind to the same ACE2 receptor [9]. Our analyses showed that pangolin-CoV-2020 has a much conserved RBD to these viruses compared to MERS-CoV, suggesting that pangolin-CoV is very likely to use ACE2 as its receptor as well. A comparative analysis of the interaction of the S proteins of coronaviruses with ACE2 proteins of humans and pangolins showed that the S proteins of SARS-CoV-2 and pangolin-CoV can potentially recognize ACE2 in both humans and pangolins [12]. A recent study found that a human ACE2-binding ridge in SARS-CoV-2 RBD takes a more compact conformation compared with the SARS-CoV RBD; moreover, several residue changes in SARS-CoV-2 RBD may also enhance its human ACE2-binding affinity [13]. The core residues in RBM which may related to higher human ACE2-binding affinity than SARS-CoV are 100% identical between SARS-CoV-2 and CoV-Pangolin-2020. Therefore, pangolin-CoV-2020 (CoV-pangolin/GD) potentially recognizes human ACE2 better than the SARS-CoV.

In addition to RBD, NTD is also important in recognizing acetylated sialic acids on glycosylated cell-surface receptors [14]. It is reported that SARS-CoV-2 can bind to human ACE2 via the viral CTD (the same as RBD), but not NTD, and that the glycan attached to Asn90 from human ACE2 forms a hydrogen bond with Arg408 from the RBD core [15]. This glycan interacting Arginine is conserved between SARS-CoV-2 and pangolin-CoV-2. Therefore, there is structural similarity in glycan binding between SARS-CoV-2 and pangolin-CoV-2020. On the other hand, ACE2 receptor is present in pangolins with a high sequence conservation with those in the gene homolog in humans. However, the zoonosis of pangolin-CoV-2020 remains unclear.

The coronaviruses are shown to have a wide range of hosts, and some of them can infect humans [16]. Thus, it is critical to determine the natural reservoir and the host tropisms of these coronaviruses, especially their potential of causing zoonosis. In the last two decades, apart from SARS-CoV-2, SARS and MERS have caused serious outbreaks in humans, leading to thousands of deaths [3, 4, 17, 18]. Although these three zoonotic coronaviruses were shown to be of bat origin, they seemed to use different intermediate hosts. For example, farmed palm civets were suggested to be an intermediate host for SARS-CoV, although the details of the link from bats to farmed palm civets remain unclear [19–21]. Most recently, dromedary camels in Saudi Arabia were shown to harbor three different coronaviruses, including the dominant MERS-CoV lineage that was responsible for the outbreaks in the Middle East and South Korea during 2015 [22]. Although this present study does not support that pangolins would be intermediate hosts for the emergence of SARS-CoV-2, our results do not exclude the possibility that other CoVs could be circulating in pangolins. Thus, surveillance of coronaviruses in pangolins could improve our understanding of the spectrum of coronaviruses in pangolins. In addition to conservation of wildlife, minimizing the exposures of humans to wildlife will be important to reduce the spillover risks coronaviruses from wild animals to humans.

In summary, we suggest that pangolins could be natural hosts of Betacoronaviruses with an unknown potential to infect humans. However, our study does not support that SARS-CoV-2 evolved directly from the pangolin-CoV.
Materials and methods
Ethics statement

The study design was approved by the ethics committee for animal experiments at the Guangdong Institute of Applied Biological Resources (reference number: GIABR20170720; 20 July 2017) and followed basic principles outlined by this committee.
Data selection

During our routine wildlife rescue efforts, one of the goals was to identify pathogens causing wildlife diseases. In 2019, we were involved in two events of pangolin rescues: one involved with 21 smuggling pangolins in March and the second with 6 smuggling pangolins in July. Although extensive rescue efforts were made, the majority of thse pangolins were dead. Most of the dead pangolins had a swollen lung, which contained a frothy liquid, and symptoms of pulmonary fibrosis. In the minority of these dead ones, we observed hepatomegaly and splenomegaly. From 11 pangolins failed to be rescued, we collected samples from their lung, lymph and spleen tissues and subjected for metagenomic analyses. Coronaviruses were detected in three individuals by mapping clean reads without ribosomes and host sequences to an in-house virus reference dataset separated from the GenBank non-redundant nucleotide database using the Burrows-Wheeler Aligner (BWA) ver 0.7.17 [10, 23]. Two of these animals were from the first batch of smuggled Malayan pangolins intercepted by Meizhou, Yangjiang, and Jiangmen customs in March, 2019, and the third one was from the second batch in a freight being transported from Qingyuan to Heyuan in July, 2019. The RNA samples from these three individuals were subjected to deep sequencing.
Genomic assembly and sequence analyses

Clean reads from each of the three coronavirus positive animals were de novo assembled using MEGAHIT v1.2.9 [24]. After examining the high similarity of 99.54% among the samples from three animals, to maximize the coverage of the virus genome, we pooled clean reads and de novo assembled them. The assembled contigs were used as references for extracting unmapped reads using Salmon v0.14.1 [25], and multiple rounds were implemented to maximize the mapping.

A total of 38 contigs were identified to be highly similar to the SARS-CoV-2 genome using BLASTn and tBLASTx. GapFiller v1.10 and SSPACE v3.0 were used to fill gaps and draft pangolin-CoV-2020 genome was constructed with ABACAS v1.3.1 (http://abacas.sourceforge.net/) [26–28].

Gaps in the draft genomes were filled using the 2x PCR Mix (Gentech, China) by reverse transcription PCR (RT-PCR). Primers were designed based on the draft genome sequence of the pangolin-CoV-2020 we assembled (S4 Table). After gap filling, the whole genome sequence of pangolin-CoV-2020 was submitted to GenBank databases (accession no. MT121216).

Multiple sequence alignments were conducted using MUSCLE [29]. Changing patterns of sequence identity were analyzed using SimPlot v3.5.1 to determine the sequence identity among SARS-CoV-2 (MN908947.3), pangolin-CoV-2020, Bat-CoV-RaTG13 (MN996532.1), Bat-CoV-ZXC21 (MG772934.1), Bat-CoV-ZC45 (MG772933.1), SARS-CoV (AY395003.1), and MERS-CoV (NC_019843.3) at both the genomic sequence level and the individual gene level [30]. The sequence identity between the whole genome and different genes or regions was calculated utilizing p-distance in MEGA v10.1.7 [31].
Phylogenetic analyses and recombination

We downloaded 44 full-length genome sequences of coronaviruses isolated from different hosts from the public database (S5 Table), with the data kindly shared by the submitters. Phylogenetic analyses were performed based on their whole genome sequences, RdRp gene, S gene, small envelope protein (E gene), as well as all other gene sequences. We constructed multiple sequence alignments of their complete genomes and individual genes using MAFFT v7.407 [32]. Phylogenetic analyses were estimated using MrBayes [33] with 50,000,000 generations and the 25% of the generations as burnin. The best models were determined by jModelTest v2.1.7 [34]. Then, the trees were visualized and exported as vector diagrams with FigTree v1.4.4 (http://tree.bio.ed.ac.uk/software/figtree/). Potential recombination events and the location of possible breakpoints in coronavirus genomes were detected using SimPlot v3.5.1 [30].
'''

title3 = "CD30-dependent degradation of TRAF2: implications for negative regulation of TRAF signaling and the control of cell survival (1997)"
example3 = '''
Abstract

CD30 is a cell-surface receptor that can augment lymphocyte activation and survival through its ability to induce the transcription factor NF-κB. CD30, however, has also been implicated in the induction of apoptotic cell death of lymphocytes. Here we show that one of the effects of CD30 signal transduction is to render cells sensitive to apoptosis induced by the type 1 tumor necrosis factor receptor (TNFR1). This sensitization is dependent on the TRAF-binding sites within the CD30 cytoplasmic domain. One of the proteins that binds to these sites is TRAF2, a signal transduction molecule that is also utilized by TNFR1 to mediate the activation of several downstream kinases and transcription factors. During CD30 signal transduction, we found that binding of TRAF2 to the cytoplasmic domain of CD30 results in the rapid depletion of TRAF2 and the associated protein TRAF1 by proteolysis. These data suggest a model in which CD30 limits its own ability to transduce cell survival signals through signal-coupled depletion of TRAF2. Depletion of intracellular TRAF2 and its coassociated proteins also increased the sensitivity of the cell to undergoing apoptosis during activation of death-inducing receptors such as TNFR1. Consistent with this hypothesis, expression of a dominant-negative form of TRAF2 was found to potentiate TNFR1-mediated death. These studies provide a potential mechanism through which CD30, as well as other TRAF-binding members of the TNFR superfamily, can negatively regulate cell survival.
Keywords

    Apoptosis NF-κB CD30 TRAF TNF proteases 

An essential feature of the immune system is the ability to tolerate large fluctuations in cell number. Homeostasis is thought to be achieved through the integration of intracellular and extracellular signals that function to balance the degree of cell proliferation with the rate of cell death (Oltvai and Korsmeyer 1994; Allison and Krummel 1995). Lymphocyte activation and proliferation is initiated by the engagement of antigen with its clonotypic cell-surface receptor (Weiss and Littman 1994). Additional costimulatory signals, however, are also required. For example, costimulatory signals can be mediated in T cells by signaling through the CD28 receptor (Allison and Krummel 1995) and in B cells through the CD40 receptor, a member of the tumor necrosis factor/nerve growth factor (TNF/NGF) receptor superfamily (Choi et al. 1995).

The TNF/NGF receptor superfamily is a group of related cell-surface receptors that include the types 1 and 2 TNF receptors (TNFR1 and TNFR2), Fas, CD27, 4-1BB, and CD30 (Vassalli 1992; Armitage 1994; Beyaert and Fiers 1994; Gruss and Dower 1995). Many of these receptors have been found to play costimulatory roles in cell proliferation, whereas others have been shown to induce apoptosis. Interestingly, several members of this family, such as TNFR2 (Tartaglia et al. 1993; Zheng et al. 1995), CD40 (Foy et al. 1996; Rathmell et al. 1996), and CD30 (see below), have been shown to play dual roles, although the mechanism(s) underlying this apparent ambiguity is unknown. To understand this phenomenon, we have examined the signaling properties of one of these receptors, CD30.

CD30 is a lymphoid cell-restricted receptor (Dürkop et al. 1992) that is normally expressed on activated and memory T cells (Ellis et al. 1993). CD30 was originally identified on the surface of Hodgkin’s lymphoma cells (Schwab et al. 1982; Stein et al. 1985b; Falini et al. 1995), and high levels of CD30 expression have also been found to correlate with disease progression in anaplastic large cell lymphoma (Nadali et al. 1995). Signal transduction through CD30 has been found to augment T-cell receptor-mediated proliferation under some circumstances (Smith et al. 1993), but in other cases to potentiate apoptosis (Lee et al. 1996). Activation of CD30 has also been shown to stimulate HIV expression (Maggi et al. 1995), probably through its ability to activate the transcription factor NF-κB (Siebenlist et al. 1994; Biswas et al. 1995). Targeted disruption of the murine CD30 gene, however, has implicated CD30 in thymic negative selection (Amakawa et al. 1996).

The signaling properties of TNFR-related receptors are being rapidly elucidated. Several TNF receptor family members, including Fas, TNFR1, and DR3/Wsl/Apo-3, have been found to contain an ∼80-residue motif designated the death domain (Tartaglia et al. 1993; Takahashi et al. 1994; Chinnaiyan et al. 1996; Kitson et al. 1996; Marsters et al. 1996; Pan et al. 1997), and signaling through these receptors has been shown to induce apoptosis in a manner that is dependent on dimerization of the death domain of the receptor with similar death domains of cytoplasmic factors such as FADD, TRADD, and RIP (Chinnaiyan et al. 1995; Hsu et al. 1995; Stanger et al. 1995). In contrast, the cytoplasmic domains of a number of other TNF receptor family members, including TNFR2 and CD40, have been shown to interact with members of the recently described TNF receptor-associated factor (TRAF) family of signaling molecules (Baker and Reddy 1996). We and others have recently identified TRAF1, TRAF2, TRAF3, and TRAF5 as factors that can be recruited to the cytoplasmic domain of CD30 and have shown that these molecules play central roles in signal transduction through this receptor (Ansieau et al. 1996; Gedrich et al. 1996; Lee et al. 1996; Aizawa et al. 1997; Duckett et al. 1997;Tsitsikov et al. 1997).

Six mammalian TRAF proteins have been identified to date (Baker and Reddy 1996). These factors have been shown to possess distinct affinities for different members of the TNF receptor superfamily. The TRAF proteins have been suggested to be important in promoting cell survival because of their role in promoting NF-κB activation through receptors of the TNF receptor family (Beg and Baltimore 1996;Liu et al. 1996; Van Antwerp et al. 1996; Wang et al. 1996). In addition, animals with a targeted TRAF3 disruption (Xu et al. 1996) and transgenic mice expressing TRAF1 (Speiser et al. 1997) have both been reported to display defects in lymphocyte development and survival.

In this report we describe a mechanism by which signaling through CD30 can both promote initial cell survival while at the same time setting in motion a signaling-dependent mechanism to leave the cell subsequently more susceptible to apoptosis through death domain-containing receptors. The ability of CD30 to promote cell survival is dependent on its ability to recruit a TRAF-dependent signal transduction complex (Duckett et al. 1997). Now, we show that the recruitment of TRAF2 into this complex results in the signal-coupled depletion of TRAF2 and its coassociated signaling molecule TRAF1. This depletion in TRAF2 correlates with increased cellular sensitivity to cell death in response to TNFR1 signal transduction. This phenomenon is not unique to CD30 signal transduction, but also occurs following TNFR2 signal transduction. Consistent with the central role of the TRAF2 signaling complex in modulating the apoptotic sensitivity of cells in response to TNFR1 engagement, dominant-negative TRAF2 was found to directly mimic the ability of CD30 signal transduction to sensitize cells to TNFR1-induced apoptosis.
Previous Section
Next Section
Results
CD30 signaling potentiates TNFR1-mediated apoptosis

Recent papers have shown an important role for NF-κB activation in the regulation of cell survival (Beg and Baltimore 1996;Liu et al. 1996; Van Antwerp et al. 1996; Wang et al. 1996), and we have shown that CD30 is a potent activator of NF-κB (Duckett et al. 1997). The human embryonic kidney cell line 293 previously has been shown to express TNFR1 but not TNFR2 (Liu et al. 1996; C.S. Duckett and C.B. Thompson, unpubl.). 293 cells were transfected with expression vectors encoding green fluorescent protein (GFP) as a marker of transfection, together with a plasmid encoding a chimeric protein comprised of the cytoplasmic domain of CD30 fused to the extracellular and transmembrane domains of CD28. This chimeric protein is expressed on the cell surface as a constitutive homodimer, and this dimerization is sufficient to initiate CD30 signal transduction in a ligand-independent manner (Duckett et al. 1997). Expression of this protein on the cell surface results in potent NF-κB activation. In contrast, a CD28 deletion mutant lacking a cytoplasmic domain (CD28ΔTail) has no effect on NF-κB activation (Duckett et al. 1997). The present experiments were initiated to determine whether CD30 signal transduction could inhibit the ability of death domain-containing proteins such as TNFR1 from inducing apoptosis. In our initial experiments, cells were stimulated with recombinant TNF-α 12 hr after transfection and subsequently evaluated for viability by nuclear staining with 4′,6′-diamidino-2-phenylindole (DAPI) and fluorescence microscopy. Contrary to our expectations, TNF-α had no effect on the viability of parental 293 cells or the cells transfected with the tailless CD28 construct. In contrast, 70% of the CD30 transfectants died in response to 18 hr of TNF-α treatment (Fig.1A). The nuclear condensation and morphological changes as observed by DAPI staining and analysis by fluorescence microscopy were indicative of apoptosis (data not shown). This cell death was dependent on TNF-α, as the CD30 chimera in the absence of TNF-α had no significant effect on viability. Flow cytometric analysis of cells transfected with the CD28–CD30 and the CD28ΔTail vectors surface expression revealed very similar levels of CD28 surface expression (data not shown). 

Previous work has shown that CD30 signal transduction is dependent on a TRAF-binding region located within the carboxy-terminal 36 amino acids of the cytoplasmic domain (Duckett et al. 1997). To determine the role of the TRAF-binding domain of CD30 in the potentiation of TNF-α-induced death, we examined the ability of a mutant of CD28–CD30 chimera that lacks the TRAF-binding motifs (ΔTraf) to potentiate TNF-α-mediated death. In contrast to the vector containing the full-length CD30 tail, this mutant was unable to induce the sensitivity to TNF-α (Fig. 1A). Thus, contrary to our expectations, these data suggest that CD30 can potentiate TNF-mediated apoptosis. This potentiation is dependent on the TRAF-binding sites in CD30.

To characterize the mechanism of cell death induced by the combination of CD30 signaling and TNF-α, expression vectors were cotransfected encoding viral proteins that are known to block apoptotic cell death induced by members of the caspase family of cysteine proteases. Expression of either the baculoviral apoptotic inhibitor, P35 (Clem et al. 1991; Bump et al. 1995), or the cowpox virus inhibitor, CrmA (Gagliardini et al. 1994; Komiyama et al. 1994), completely abrogated the cell death induced by TNF-α plus CD30 signaling (Fig. 1B), indicating the involvement of caspases and confirming that the cell death was caused by apoptosis.
TRAF2 is degraded upon recruitment to CD30

The experiments described above suggested a role for the TRAF proteins in the potentiation of TNFR1-mediated death. Because TRAF2 has also been found to be involved in TNFR1-mediated signaling (Hsu et al. 1996), we sought to characterize cellular TRAF2. Endogenous TRAF2 protein in 293 cells was not detectable by immunoblot analysis with commercially available TRAF2-specific antibodies (Fig.2A; data not shown), indicating that TRAF2 is normally expressed at levels below the threshold of detection in this cell type. Therefore, a mammalian expression vector encoding human TRAF2 was transfected into 293 cells together with either the chimeric CD28 vector containing the CD30 cytoplasmic tail, or the CD28 vector lacking a cytoplasmic tail. Surprisingly, whereas TRAF2 protein was easily detected in lysates from cells cotransfected with the TRAF2 plus control CD28 tailless vectors, TRAF2 was virtually undetectable when cotransfected with the CD28–CD30 chimera (Fig. 2A). This effect was not observed when TRAF2 was coexpressed with a CD30 vector lacking the TRAF-binding domains. High levels of TRAF2 protein were also detected when the TRAF2 vector was transfected alone or with a variety of additional control vectors (data not shown). 

To investigate why we failed to detect TRAF2 protein in cells transfected with TRAF2 in the presence of the chimeric CD28–CD30 receptor, the levels of TRAF2 RNA were examined by Northern analysis. The amounts of TRAF2 RNA were similar in cells cotransfected with the CD28ΔTail, CD28–CD30, and CD28–CD30ΔTraf vectors (Fig. 2B). These data suggest that TRAF2 is being transcribed at equivalent rates in the presence or absence of the CD28–CD30 receptor. The failure of TRAF2 protein to accumulate appears to be mediated by post-transcriptional mechanisms.
CD30-induced destabilization of TRAF2 can be blocked by protease inhibitors

To further characterize the process of CD30-induced TRAF2 destabilization, a panel of protease inhibitors were tested for their ability to inhibit CD30-induced TRAF2 degradation (Fig.3). Lactacystin, a specific inhibitor of the ubiquitin-proteasome pathway (Krappmann et al. 1996), was unable to rescue the degradation of TRAF2 (Fig. 3), although a number of less specific compounds such as E64 and TLCK were able to inhibit TRAF2 degradation. In addition, several inhibitors of the calpain family of cysteine proteases were found to efficiently block CD30-induced TRAF2 degradation. The calpains are a family of cytoplasmic, calcium-activated cysteine proteases that have been implicated in the induction of apoptosis (Squier and Cohen 1997). These data suggest that the binding of TRAF2 to the cytoplasmic tail of CD30 activates a protease or proteases that result in the degradation of TRAF2. 

Heteromeric complexes containing TRAF2 are also degraded upon recruitment to CD30

It has previously been shown that TRAF1, TRAF2, and TRAF3 can be directly recruited to the cytoplasmic domain of CD30 (Ansieau et al. 1996; Gedrich et al. 1996; Lee et al. 1996). More recently, three additional members of the TRAF family have also been identified, TRAF4, TRAF5, and TRAF6 (Tomasetto et al. 1995; Cao et al. 1996; Ishida et al. 1996a,b; Nakano et al. 1996; Aizawa et al. 1997). Therefore, the six known TRAF proteins were assessed for their abilities to be degraded upon CD30 signaling (Table 1). Interestingly, TRAF2, TRAF3, and TRAF5 were degraded, presumably upon binding to CD30, whereas the levels of TRAF1, TRAF4, and TRAF6 were not evidently affected by coexpression of CD30 (Table 1). TRAF1 is able to bind to the CD30 tail, but was not degraded, suggesting that additional determinants may be present in TRAFs 2, 3, and 5 which confer binding-specific degradation. 

As shown in Table 1, TRAF1 was not degraded by CD30 signal transduction. Because TRAF1 and TRAF2 can form heterodimers (Rothe et al. 1994, 1995b), we tested the possibility that TRAF1 might function to stabilize TRAF2. Expression vectors encoding TRAF1 and TRAF2 were cotransfected into 293 cells together with either the CD28–CD30 vector or the control vector lacking a cytoplasmic tail. Cotransfection of TRAF1 had no stabilizing effect on TRAF2 (Fig. 4A). In fact, whereas expression of TRAF1 alone was not affected by the presence of CD30, coexpression of TRAF1 with TRAF2 resulted in the almost complete disappearance of TRAF1, suggesting that recruitment by TRAF2 of TRAF1 into a heteromeric complex with CD30 resulted in degradation of both TRAF proteins. 

The RING finger of TRAF2 is required for CD30-mediated degradation

Both TRAF1 and TRAF2 possess similar characteristics of recruitment to the two TRAF-binding sites in CD30 (Gedrich et al. 1996), but as shown in Figure 4A, TRAF1 was not degraded directly by CD30 signal transduction. The principal structural difference between TRAF1 and TRAF2 is the presence of a RING finger domain in TRAF2. Therefore, the properties of a mutant version of TRAF2 that lacks the RING finger domain were examined. Deletion of the amino-terminal 86 residues of TRAF2 has previously been reported to disrupt the RING finger domain of TRAF2 without diminishing its ability to bind to CD30 (Gedrich et al. 1996). This mutant, TRAF2 (87–501), functions in a dominant-negative manner to inhibit CD30-mediated signaling (Duckett et al. 1997). As shown in Figure 4B, TRAF2 (87–501) was not degraded by coexpression with CD30, whereas in control transfections, the level of full-length TRAF2 was dramatically reduced. These findings suggest that the amino terminus of TRAF2, which contains a RING finger, is required for degradation following recruitment to the cytoplasmic tail of CD30.

Previous studies have shown that TRAF2 is able to form homodimers, as well as heterodimers with TRAF1 (Rothe et al. 1995b). Because TRAF1 was degraded when coexpressed with TRAF2 and CD30, we reasoned that the dominant-negative TRAF2 (87–501), which is not degraded when coexpressed with CD30 (Fig. 4B), might become subject to degradation if coexpressed with full-length TRAF2. As shown in Figure 4B, levels of cotransfected full-length TRAF2 and truncated TRAF2 (87–501) were dramatically reduced on coexpression with the CD28–CD30 chimera. This suggests that a complex containing both the full-length and the truncated (87–501) forms of TRAF2 is degraded upon recruitment to CD30.
Both TRAF-binding sites in CD30 contribute to TRAF degradation

Previously we have shown that TRAF2 can bind independently to either of two 5–7 residue elements in CD30 (domains 2A and 2B), and that each of these elements are independently capable of recruiting TRAF proteins and inducing NF-κB activation (Gedrich et al. 1996;Duckett et al. 1997). Therefore, additional mutants of the CD30 cytoplasmic tail were tested to determine which of the two TRAF-binding elements was required to produce destabilization of TRAF2. The TRAF2 expression vector was cotransfected into 293 cells together with chimeric CD30 variants lacking the TRAF-binding domain 2A only, 2B only, or both 2A and 2B. Individual disruption of either TRAF-binding domain 2A or 2B resulted in mutants that induced intermediate levels of TRAF2 degradation (Fig. 5). In control transfections, wild-type CD30 resulted in the almost complete loss of TRAF2. These data suggest that whereas either TRAF-binding site is capable of partially destabilizing TRAF2, both sites are required for maximal destabilization. 

TNFR2 can also enhance TNFR1 sensitivity and induce TRAF2 degradation

To determine whether the ability to heighten sensitivity to TNFR1 signals and to induce the degradation of TRAF2 is a unique property of CD30 or is shared by other TNF receptor family members, we examined the properties of TNFR2. A chimeric vector was constructed encoding the cytoplasmic domain of TNFR2 fused to the extracellular and transmembrane domains of CD28. Transfection of this vector into 293 cells resulted in comparable levels of both surface CD28 expression and NF-κB induction to those observed after transfection of the CD28–CD30 chimera or the control CD28Δtail vector (data not shown). Similar to the results found with the CD28–CD30 protein, cells transfected with the CD28–TNFR2 chimera were also sensitized to TNFR1-mediated death. The degree of sensitization was almost identical to that found for the CD28–CD30 chimera (Fig. 6A). In addition, cotransfection of CD28–TNFR2 with TRAF2 also induced the degradation of TRAF2 (Fig. 6B), revealing that the enhancement of TNFR1 sensitivity, as well as the TRAF2 degradation following recruitment to the receptor, is not restricted to CD30, but can occur with other members of the TNF receptor superfamily. 
'''

examples = [
    example1,
    example2,
    example3
]

titles = [
    title1,
    title2,
    title3,
]
